### ISSUE TITLE
iOS - Bookmark Screen - VoiceOver Does Not Tell User which Item They are Attempting to Remove

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#150]
When a user attempts to remove a bookmark VoiceOver does not tell the user which item/bookmark they are attempting to remove, There should be some type of off-screen text or notification of the item being removed. An example would be "Remove XYZ bookmark" instead of "Remove bookmark", where XYZ is the page of the screen to be removed.

There is a similar issue for web-based pages and Android that would need to be corrected too.

**Comment by jfc3-dol** [comment]
TalkBack is still reading  "Remove bookmark" instead of "Remove XYZ bookmark", where XYZ is the page or the screen to be removed.

**Comment by jfc3-dol** [comment]
@pbhatt17 

I tested the remove bookmark on Android, iOS, and the web. Now the screen reader's read which items the person is going to remove.

So I'm going to close this issue.

NOTE - I did notice that when issuing Android and need to check back on the other that TalkBack is reading the text before the "Remove" button twice. Once as a link and the second as a button or vice versa. So will open a new issue if there does not already have one that exists.
